app/views/helpers/tax_helper.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class TaxHelper
+  def self.applied_taxes(object)
+    slim_template = <<-SLIM_TEMPLATE
+- (applied_taxes.present? ? applied_taxes.pluck(:tax_rate) : [0.0]).each do |tax|
```

Comment:
Layout/HeredocIndentation: Use 2 spaces for indentation in a heredoc by using `<<~` instead of `<<-`. (https://rubystyle.guide#squiggly-heredocs)